flink-dist/src/main/flink-bin/bin/taskmanager.sh

```diff
@@ -73,6 +73,13 @@ if [[ $STARTSTOP == "start" ]]; then
                 TM_OFFHEAP_SIZE=${FLINK_TM_MEM_MANAGED_SIZE}
                 TM_HEAP_SIZE=$((FLINK_TM_HEAP - FLINK_TM_MEM_MANAGED_SIZE))
             else
+                # Bash only performs integer arithmetic so floating point computation is performed using bc
+                BC_PATH=`command -v bc`
+                if [[ $? -eq 1 || ! -f $BC_PATH ]]; then
```

Comment:
I think this may cause problems when `bc` is symlinked. The error code of `command -v bc` should be enough.